manifests/etcd/0.9.2/etcdoperator.v0.9.2.clusterserviceversion.yaml

```diff
@@ -4,6 +4,9 @@ kind: ClusterServiceVersion
 metadata:
   name: etcdoperator.v0.9.2
   namespace: placeholder
+  labels:
+    olm.label: testlabel
+    olm.label.1: testlabel1
```

Comment:
Should we be putting invalid values into the canonical examples in our `/manifests` folder?

Reply:
All of the test cases that are for property extraction using manifests from this folder so that's why I added it here. This is just another folder for test manifests. I don't see an issue with changing. Plus, additional properties to this operator is harmless given it only matters if someone requires this property.

Reply:
It feels to me like the `/manifests` folder should be a good example of what manifests should be, and if we want something with invalid data, it should be saved somewhere else.

Reply:
The labels above are totally valid.

Reply:
Also, just a note. The manifests inside `/manifests` are hardly good examples given one of them has multiple CSV under same directory and one operator uses flatten format that we don't use anymore. The manifests in `/bundles` are actually better as examples.